server_admin/topics/admin-console-permissions/fine-grain.adoc

```diff
@@ -43,20 +43,20 @@ The first thing we must do is login to the Admin Console so we can set up permis
 of the client, we want to define fine-grain permissions for.
 
 .Client management
-image:{project_images}/fine-grain-client.png[]
+image:{project_images}/fine-grain-client.png[Fine grain client]
 
 You should see a tab menu item called `Permissions`.  Click on that tab.
 
 .Client permissions tab
-image:{project_images}/fine-grain-client-permissions-tab-off.png[]
+image:{project_images}/fine-grain-client-permissions-tab-off.png[Fine grain client permissions tab]
 
 By default, each client is not enabled to do fine grain permissions.  So turn the `Permissions Enabled` switch to on
 to initialize permissions.
 
 IMPORTANT: If you turn the `Permissions Enabled` switch to off, it will delete any and all permissions you have defined for this client.
 
 .Client permissions tab
-image:{project_images}/fine-grain-client-permissions-tab-on.png[]
+image:{project_images}/fine-grain-client-permissions-tab-on.png[Fine grain permission tab]
```

Comment:
Text on this screen is hard to read.  Can you shrink this screen by a quarter and recapture it?

Reply:
The others look good, but this one could still be resized.